docs/manual/forms/variables.rst

```diff
@@ -0,0 +1,170 @@
+
+.. note:: Het is een nieuwere en volledigere variant van de ``auth`` variabele.
+   Er zijn geen plannen om deze laatste te verwijderen.
+
+De variabele bevat een bak aan informatie, gestructureerd volgens het
+authenticatiecontextdatamodel (TODO: add link). De structuur is als volgt:
+
+.. code-block:: json
+
+    {
+        "source": "string",
+        "levelOfAssurance": "string",
+        "representee": {
+            "identifierType": "string",
+            "identifier": "string"
+        },
+        "authorizee": {
+            "legalSubject": {
+                "identifierType": "string",
+                "identifier": "string"
+            },
+            "actingSubject": {
+                "identifierType": "string",
+                "identifier": "string"
+            }
+        },
+        "mandate": {
+            "role": "string",
+            "services": [
+                {
+                    "id": "string",
+                    "uuid": "string"
+                }
+            ]
+        }
+    }
+
+Merk op dat niet alle attributen aanwezig zijn, dit hangt af van het inlogmiddel (
+DigiD, eHerkenning) en of er wel/niet sprake is van een machtiging én de soort
+machtiging.
+
+De volgende attributen zijn gegarandeerd aanwezig:
+
+* ``source``, mogelijk lege string als waarde
+* ``levelOfAssurance``, mogelijk lege string als waarde
+* ``authorizee``
+* ``authorizee.legalSubject``
+* ``authorizee.legalSubject.identifierType``, mogelijk lege string als waarde
+* ``authorizee.legalSubject.identifier``, mogelijk lege string als waarde
+
+De onderdelen van deze structuur worden ook als individuele variabelen aangeboden:
+
+``auth_context_source``
+    Middel van inloggen: de waarde is ``"digid"`` of ``eherkenning``, of een lege string
+    wanneer er geen informatie beschikbaar is.
+
+``auth_context_loa``
+    Betrouwbaarheidsniveau waarmee ingelogd is. Kan leeg zijn indien onbekend.
+
+``auth_context_representee_identifier_type``
+    Geeft aan of het om een BSN of KVK-nummer gaat, en bepaalt dus de soort
+    vertegenwoordigde. Leeg indien onbekend of als er geen sprake is van machtigen.
+
+``auth_context_representee_identifier``
+    Identificatie van de vertegenwoordigde. Leeg indien onbekend of als er geen sprake
+    is van machtigen.
+
+``auth_context_legal_subject_identifier_type``
+    Geeft aan of het om een BSN of KVK-nummer gaat, en bepaalt dus de soort
+    (wettelijke) vertegenwoordigder (of de ingelogde persoon zelf indien er geen sprake
+    is van vertegenwoordiging). Leeg indien het formulier zonder inloggen gestart is.
+
+``auth_context_legal_subject_identifier``
+    Identificatie van de (wettelijke) vertegenwoordiger. Leeg indien het formulier
+    zonder inloggen gestart is.
+
+``auth_context_acting_subject_identifier_type``
+    In de praktijk zal de waarde altijd ``opaque`` of leeg zijn. Geeft aan hoe de
+    identificatie van de handelende persoon ("de persoon aan de knoppen")
+    geïnterpreteerd moet worden.
```

Comment:
I'm not sure what this means, does this mean that the value literally is `"opaque"`? Is this like a convenience variable to not have to check if machtigen is used when determining the acting subjects auth info?

Reply:
the identifier type value is indeed `"opaque"`, as opposed to `"bsn"` or `"kvkNummer"` (or the future `"RSIN"`).

Currently it's the only possible type, but in the future more types could be possible (this could for example be used for a registrator at the service desk too, at some point). It basically means that you have zero guarantees about the meaning of the identifier value except that it's a value tied to a particular staff member. It *could* look like a BSN, but if you use it to consult the BRP, it may sometimes work, it will most likely not. It is not expected to be decrypted.